code/components/debug-net/src/NetDebug.cpp

```diff
@@ -32,12 +33,11 @@ const int g_netOverlaySampleCount = 150;
 class RageHashList
 {
 public:
-	template<int Size>
-	RageHashList(const char* (&list)[Size])
+	RageHashList()
 	{
-		for (int i = 0; i < Size; i++)
+		for (const auto packet : g_knownPackets)
 		{
-			m_lookupList.insert({ HashRageString(list[i]), list[i] });
+			m_lookupList.insert({ packet.first,  packet.second });
 		}
 	}

```

Comment:
Seeing our lists are the same now, we can just refer to the original and not make a copy